### ISSUE TITLE
.wacz signature contains incorrect `SubSeqMaxOffset`

### ISSUE THREAD

**Issue by rien333** [digital-preservation/droid#1067]
## Issue description ##

The [most recent container signature file](https://cdn.nationalarchives.gov.uk/documents/container-signature-20231127.xml) identifies `.wacz` files based on the following signature:

```xml
<ContainerSignature Id="80000" ContainerType="ZIP">
      <Description>Web Archive Collection Zipped</Description>
      <Files>
        <File>
          <Path>datapackage.json</Path>
          <BinarySignatures>
            <InternalSignatureCollection>
              <InternalSignature ID="80000">
                <ByteSequence Reference="BOFoffset">
                  <SubSequence Position="1" SubSeqMinOffset="0" SubSeqMaxOffset="4096">
                    <Sequence>'wacz_version'</Sequence>
                  </SubSequence>
                </ByteSequence>
                 ...
```

However,  the crawler [browsertrix](https://github.com/webrecorder/browsertrix-crawler) — developed by exact the same people as the `.wacz` file format — puts the `wacz_version` string all the way at the bottom of the embedded `datapackage.json` file. As such, programs that use this container signature file will currently incorrectly identify browsertrix's `.wacz` files as `.zip` files.  

Indeed, the `.wacz` [specification](https://specs.webrecorder.net/wacz/1.1.1/#datapackage-json) nowhere mandates that the `wacz_version` string should be present at _the top_ of the `datapackage.json` file.  From what I can see, it can be present anywhere in the file.

In short, all this means that pronom detection will often fail on fairly standard `.wacz` files, namely those produced by what might  well be the only crawler that outputs web archives in this format. 

## Proposed solution ##

Changing `SubSeqMaxOffset` to a higher number (e.g. 324.096) fixes the signature-based detection. Alternatively, it might be wise to remove this `SubSeqMaxOffset` altogether, insofar this is possible.

## Additional info

Example `.wacz`  file:  [browsertrix-webcurator-org.wacz.zip](https://github.com/digital-preservation/droid/files/14241384/browsertrix-webcurator-org.zip)

**Comment by Dclipsham** [comment]
There's a fix in-hand for this - it changes the sequence to a fully variable one within the datapackage.json file so the whole file will be scanned - I've been told it'll be in the next PRONOM update:
```
<ContainerSignature Id="80000" ContainerType="ZIP">
  <Description>Web Archive Collection Zipped</Description>
  <Files>
    <File>
      <Path>datapackage.json</Path>
      <BinarySignatures>
        <InternalSignatureCollection>
          <InternalSignature ID="80000">
            <ByteSequence Reference="Variable">
              <SubSequence Position="1">
                <Sequence>'wacz_version'</Sequence>
              </SubSequence>
            </ByteSequence>
          </InternalSignature>
        </InternalSignatureCollection>
      </BinarySignatures>
    </File>
  </Files>
</ContainerSignature>
```

David

**Comment by rien333** [comment]
All right, I will close this for now then!
